livekit_room.proto

```diff
@@ -39,6 +39,9 @@ service RoomService {
 
   // Send data over data channel to participants in a room, Requires `roomAdmin`
   rpc SendData(SendDataRequest) returns (SendDataResponse);
+
+  // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
+  rpc UpdateRoomMetadata (UpdateRoomRequest) returns (Room);
```

Comment:
```suggestion
  rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
```

Reply:
it should returns UpdateRoomMetadataResponse